cmd/vic-machine/create/create.go

```diff
@@ -485,7 +485,7 @@ func (c *Create) Flags() []cli.Flag {
 
 		cli.BoolFlag{
 			Name:        "force, f",
-			Usage:       "Force the install, removing existing if present",
```

Comment:
@morris-jason so turns out I had a misunderstanding 😄  `--force` does not install over existing installments. Looks great!